modules/auxiliary/scanner/http/simplehelp_toolbox_path_traversal.rb

```diff
@@ -0,0 +1,109 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::Scanner
+  include Msf::Exploit::Remote::HttpClient
+
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'SimpleHelp Path Traversal Vulnerability CVE-2024-57727',
+        'Description' => %q{
+          There exists a path traversal vulnerability in the /toolbox-resource endpoint that enables unauthenticated
+          remote attackers to download arbitrary files from the SimpleHelp server via crafted HTTP requests
+        },
+        'Author' => [
+          'horizon3ai', # discovery
+          'imjdl',      # CVE-2024-57727 PoC
+          'jheysel-r7'  # module
+        ],
+        'References' => [
+          [ 'URL', 'https://www.horizon3.ai/attack-research/disclosures/critical-vulnerabilities-in-simplehelp-remote-support-software/'], # Discovery
+          [ 'URL', 'https://simple-help.com/kb---security-vulnerabilities-01-2025#security-vulnerabilities-in-simplehelp-5-5-7-and-earlier'], # Vendor Advisory
+          [ 'URL', 'https://rustlang.rs/posts/simple-help/'], # PoC for Path Traversal CVE-2024-57727
+          [ 'URL', 'https://attackerkb.com/topics/G4CTOrbDx0/cve-2024-57727'], # PoC for Path Traversal CVE-2024-57727
+          [ 'CVE', '2024-57727'],
+        ],
+        'License' => MSF_LICENSE,
+        'DisclosureDate' => '2025-01-12',
+        'Notes' => {
+          'Stability' => [ CRASH_SAFE, ],
+          'SideEffects' => [ ARTIFACTS_ON_DISK, ],
```

Comment:
What are the artifacts left behind by this module?

Reply:
This has been updated to be `IOC_IN_LOGS`.